config.json

```diff
@@ -1,24 +0,0 @@
-{
```

Comment:
Rather than removing `config.json`, perhaps rename it to `config.json.default`?

I don't have Diablo II installed on my system, so trying to locate the MPQ archives in the *standard location* would fail. As such, I would use a custom `config.json` to run OpenDiablo2, and having an example config file is always useful.

Reply:
Ah I agree. I will restore.